1. CWE Base Mod/Cold War Era Mod (CWE)/common/scripted_effects/00_building_setup.txt

```diff
@@ -9,7 +9,7 @@ effect_starting_buildings_1 = {
 	#	}
 		create_building = {
 			building = "building_wheat_farm"
-			level = 20
+			level = 50
```

Comment:
I think it might be good to increase the work force requirement for wheat farms instead of increasing the building level of the building level is to be increased by a lot. This is so that there’s no need to re balance arable land per state vales.

Reply:
Commit 66231b4 solves this